tiny-firmware/firmware/fsm_skycoin.c

```diff
@@ -86,7 +87,9 @@ void fsm_msgSkycoinSignMessage(SkycoinSignMessage *msg) {
     ResponseSkycoinAddress respAddr;
     uint8_t seckey[32] = {0};
     uint8_t pubkey[33] = {0};
-    ErrCode_t err = fsm_getKeyPairAtIndex(1, pubkey, seckey, &respAddr, msg->address_n);
+    int (*address_from_pubkey)(const uint8_t*, char*, size_t*);
+    address_from_pubkey = &skycoin_address_from_pubkey;
+    ErrCode_t err = fsm_getKeyPairAtIndex(1, pubkey, seckey, &respAddr, msg->address_n, address_from_pubkey);
```

Comment:
You can omit creating `address_from_pubkey` variable and directly path `&skycoin_address_from_pubkey` to fsm_getKeyPairAtIndex.